src/scenes/UserGalleryPage/UserGalleryCollection.tsx

```diff
@@ -13,37 +14,44 @@ type Props = {
   collection: Collection;
 };
 
-// Space between NFTs in pixels
-const GAP_PX = 40;
+export function isValidColumns(columns: number) {
+  return columns >= 1 && columns <= 6;
```

Comment:
hypernit: make 1 and 6 constants MIN_COLS, MAX_COLS

Reply:
done